### PR TITLE
markdown に texてきな感じで数式を。

### PR DESCRIPTION
**Pull request by sosuke-k**
## Why
- `marked.js` が数式に対応してしない
- マークダウンのプレビューエリアにマージンがない
## What

[Gyazo](http://gyazo.com/0610fc625ad348d178c030038cd8cc08)
## How
- [MathJax](https://www.mathjax.org/)を導入
- [MathJaxの使い方](http://genkuroki.web.fc2.com/)
- [How to use Markdown with MathJax like Math StackExchange](http://stackoverflow.com/questions/12069625/how-to-use-markdown-with-mathjax-like-math-stackexchange)
- プレビューエリアに `padding` 追記
## <(_ _)>
- `editor` -> `marked` -> `MathJax` -> `preview` なので
-  `_` など一行で２回以上使うと `marked` によって中が変換されてしますようなものは、`\_` としてエスケープしないとダメ。
- `\` も `\\` としないとダメ。
- `code` ないではダメ。
- 行数が変わらないと `MathJax` のレンダーが発火しない？

### CODE REVIEW

**Comment by sosuke-k** [comment]
@mondamin おなしゃす
